make/mt-os/mt-stm32fx.mk

```diff
@@ -90,6 +90,7 @@ endif
 XI_CONFIG_FLAGS += -DXI_CROSS_TARGET
 XI_CONFIG_FLAGS += -DXI_EMBEDDED_TESTS
 
+XI_COMPILER_FLAGS += -DLWIP_TIMEVAL_PRIVATE=0
```

Comment:
very good catch! This is not and SFT issue. This was an old issue back when I did the port on STM32 Eth which uses LWIP as network stack. timeval is defined in stdio.h and in LWIP somewhere (maybe netdb.h ?). And it conflits. Which resulted in the inability of emitting debug output in code which is compiled against STM32 Eth. And this has not only effect on STM32 but common code as well. And I faced with this problem too: When I left accidentally (or intentionally) a printf in a common code then the STM32 eth build failed with: conflicting defines on timeval.

And no I found that timeval def can be tuned of in LWIP. So this is a "side" maintenance change.